source/endpoint-reference/endpoint-folders.rst

```diff
@@ -0,0 +1,191 @@
+                 each of the objects by looking at the tuple's `type` attribute.
+                 Folders will have type "folder" while variables will contain
+                 their variables type.
+-------- ------- ---------------------------------------------------------------
+graph    list    A flat list containing the list of URLs of variables and
+                 subfolders on the given folder.
+======== ======= ===============================================================
+
+JSON example:
+
+.. language_specific::
+   --JSON
+   .. code:: json
+
+      {
+          "element": "shoji:catalog",
+          "self": "https://app.crunch.io/api/datasets/5ee0a0/variables/",
+          "size": 10000,
+          "body": {
+              "name": "Folder name"
+          },
+          "index": {
+              "https://app.crunch.io/datasets/abcdef/variables/123/": {
+                  "name": "Birth Year",
+                  "derived": false,
+                  "discarded": false,
+                  "alias": "birthyear",
+                  "type": "numeric",
+                  "id": "123",
+                  "notes": "",
+                  "description": "In what year were you born?"
+              },
+              "https://app.crunch.io/datasets/abcdef/folders/qwe/": {
+                  "name": "Subfolder name"
+              },
+              "https://app.crunch.io/datasets/abcdef/variables/456/": {
+                  "subvariables_catalog": "aad4ad/subvariables/",
+                  "name": "An Array",
+                  "derived": true,
+                  "discarded": false,
+                  "alias": "arrayvar",
+                  "subvariables": [
+                      "439dcf/",
+                      "1c99ea/"
+                  ],
+                  "notes": "All variable types can have notes",
+                  "type": "categorical_array",
+                  "id": "456",
+                  "description": ""
+              }
+          },
+          "graph": [
+              "https://app.crunch.io/datasets/abcdef/variables/456/",
+              "https://app.crunch.io/datasets/abcdef/variables/123/",
+              "https://app.crunch.io/datasets/abcdef/folders/qwe/"
+          ]
+      }
+
+
+Creating folders
+----------------
+
+To create a subfolder, clients must POST to the target folder (which will be the
+parent of the new subfolder).
+
+The payload must contain a `body` member which indicates only the name of
+the subfolder. Note that subfolder names must be unique between them and
+variables included.
+
+Additionally a `graph` member can be included that must contain URLs of the
+varibles that will be children of the new folder. These variables cannot belong
+to another folder (must be ungrouped under root) else the server will return
+ a 400 response.
+To include variables that belong to other folder in the new folder they
+should be moved into the new folder after the fact via PATCH.
```

Comment:
Why?

Reply:
I... don't remember why. I think it was an early imlpementation issue that certainly doesn't stand anymore.

I will address this and the comment above, where items need to be included from the index. 
https://www.pivotaltracker.com/story/show/154267423